Gemfile

```diff
@@ -26,9 +26,10 @@ group :development do
   gem "puppet-module-win-dev-r#{minor_version}", '~> 1.0',       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-resource_api",                                     require: false
   gem "github_changelog_generator",                              require: false
-  gem 'retriable', '~> 3.1',                                     require: false
-  gem 'pry-byebug',                                              require: false
-  gem 'webmock',                                                 require: false
+  gem "ruby-pwsh",                                               require: false
+  gem "webmock",                                                 require: false
+  gem "pry-byebug",                                              require: false
```

Comment:
Is pry-byebug necessary ?

Reply:
sure, will remove it. It was just for debugging. Thank you @carabasdaniel